ui/app/src/Utils/Api.elm

```diff
@@ -66,7 +66,7 @@ iso8601Time =
 
 baseUrl : String
 baseUrl =
-    "/api/v1"
+    "http://localhost:9093/api/v1"
```

Comment:
Both in the dev-server and the actual Alertmanager. I would like to not have two `index.html` files.

Happy for suggestions. Definitely a thing we can improve.